rest/src/main/kotlin/builder/components/CompositeActionRowBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package dev.kord.rest.builder.components
+
+import dev.kord.common.annotation.KordPreview
+
+@KordPreview
+interface CompositeActionRowBuilder {
+    fun actionRow(builder: CompositeButtonBuilder.() -> Unit)
+}
```

Comment:
Making this non-inline prevents people from calling suspending functions in the lambda, we generally want to avoid that.

Additionally, there's no way to access, modify or delete any of the already created buttons/rows.